src/redux/books/BookSlice.js

```diff
@@ -0,0 +1,21 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  books: [],
+};
+
+const bookSlice = createSlice({
+  name: 'books',
+  initialState,
+  reducers: {
+    addBook: (state, { payload }) => {
+      state.books.push({ title: payload.title, author: payload.author });
```

Comment:
- [ ] Kindly consider adding an `id` property, so when you try to remove the book you can select it by the id but in your case, the book doesn't have any book.  You can use a random method like this `  id: Math.floor(Math.random() * 100),`